RecoHGCal/TICL/plugins/PatternRecognitionbyCLUE3D.cc

```diff
@@ -454,17 +454,17 @@ void PatternRecognitionbyCLUE3D<TILES>::calculateLocalDensity(
 
   for (unsigned int i = 0; i < numberOfClusters; i++) {
     // We need to partition the two sides of the HGCAL detector
-    auto lastLayerPerSide = static_cast<unsigned int>(rhtools_.lastLayer(false)) - 1;
-    unsigned int minLayer = 0;
-    unsigned int maxLayer = 2 * lastLayerPerSide - 1;
+    auto lastLayerPerSide = static_cast<int>(rhtools_.lastLayer(false));
```

Comment:
This is not the same as previous `lastLayerPerSide`, and `maxLayer` as well as a consequence: please confirm that this is wanted

Reply:
Ciao @perrotta that's correct, the `-1` was a bug as well.